### PR TITLE
RTS-704 timeseries list keys

### PR DESCRIPTION
**Pull request by bkerley**

### CODE REVIEW

File: spec/integration/riak/time_series_spec.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@
 describe 'Time Series', test_client: true, integration: true do
   let(:table_name){ 'GeoCheckin' }
 
-  let(:now){ Time.now }
-  let(:fiveMinsAgo){ Time.now - 300 }
+  let(:now){ Time.at(Time.now.to_i) }
```
Comment (lukebakken) [inline spec/integration/riak/time_series_spec.rb]: Why did this change from just `Time.now`?

File: spec/integration/riak/time_series_spec.rb [hunk 2]
```diff
@@ -4,8 +4,8 @@
...
+  let(:fiveMinsAgo){ Time.at((Time.now - 300).to_i) }
```
Comment (lukebakken) [inline spec/integration/riak/time_series_spec.rb]: Should this be:

`let(:fiveMinsAgo){ now - 300 }`

**Comment by lukebakken** [comment]
Just a couple comments, then :shipit:

File: spec/integration/riak/time_series_spec.rb (hunk 1 above)
Comment (bkerley) [inline spec/integration/riak/time_series_spec.rb]: Ruby `Time` objects and `Time.now` have higher precision than Riak TS. By truncating them to seconds on initialization, I avoid having to do complex rounding when checking to see if data returning from Riak has the matching time.

File: spec/integration/riak/time_series_spec.rb (hunk 2 above)
Comment (bkerley) [inline spec/integration/riak/time_series_spec.rb]: Yeah, that'd do it :P

File: spec/integration/riak/time_series_spec.rb (hunk 1 above)
Comment (lukebakken) [inline spec/integration/riak/time_series_spec.rb]: OK I figured it was something clever like that :smile:

**Comment by lukebakken** [comment]
+1 dbd5bc5be8e29dccf469bda6f2a7f20c1e9dc661

**Pull request merged**